### ISSUE TITLE
Cannot create events while offline

### ISSUE THREAD

**Issue by LeonStaufer** [marcoancona/MineTime#152]
When trying to create an event with no active internet connection, MineTime displays the error message that this is impossible.

It would be nice to add the option to create events offline, that then sync once online again.

**Comment by marcoancona** [comment]
Agree, this is in our roadmap since a while but will require some major changes so I am not sure when the offline-mode will be available.
